update_db.php

```diff
@@ -0,0 +1,37 @@
+<?php
+require 'functions.php';
+$db = get_db();
+$winner = $_POST;
+test_input();
+add_winner($db, $winner);
+/*
+* Inserts new entry into the DB
+*
+* @param object DB of previous winners
+* @param validated array
+*
+* @return
+*/
+function add_winner(object $db, array $winner)
+{
+    if (!empty($_POST)) {
+        if (($winner['prize_year'] > 1968 && $winner['prize_year'] <= date("Y"))
+            && ($winner['author_name'] != "" && $winner['book_name'] != "" && $winner['author_nationality'] != "")
+            && (strlen($winner['author_name']) > 6 && strlen($winner['author_name']) < 200)
+            && (strlen($winner['book_name']) > 6 && strlen($winner['book_name']) < 300)
+            && (strlen($winner['author_nationality']) > 2 && strlen($winner['author_nationality']) < 50)) {
```

Comment:
Since you're having to repeat this logic it could be worth putting it into a dedicated validation  function you could re-use